project.clj

```diff
@@ -16,12 +16,13 @@
 
   :dependencies [[org.clojure/clojure]
                  [org.clojure/tools.logging]
+                 [org.clojure/tools.nrepl "0.2.13"]
```

Comment:
Why is this needed now?

Reply:
It resolves a conflict between leiningen and clj-parent 2.x, I think every project that has gone to clj-parent 2.x has had to do this.

Reply:
Yeah I've seen that elsewhere, just think I missed what it was for. Thanks!